.travis.yml

```diff
@@ -1,126 +1,177 @@
 # This Travis job script has been generated by a script via
 #
-#   runghc make_travis_yml_2.hs '--config' '--output=.travis.yml' 'cabal.project'
+#   haskell-ci 'cabal.project' '--output' '.travis.yml'
```

Comment:
if you rename `cabal.make-travis-yml` to `cabal-haskell-ci` (which is a convention) and use `haskell-ci --config=cabal.haskell-ci cabal.project`, you'll pick options from there. E.g. `branches`.

Reply:
Thanks, good to know 👍